src/promptflow-core/promptflow/executor/flow_executor.py

```diff
@@ -62,12 +62,7 @@
 from promptflow.tracing._integrations._openai_injector import inject_openai_api
 from promptflow.tracing._operation_context import OperationContext
```

Comment:
Maybe add a test case for the scenario mentioned in the description?

Reply:
updated the fix way, and add a test in start trace UT, may take a look